README.md

```diff
@@ -11,7 +22,13 @@ This tool can be used as a workaround for https://github.com/aspnet/EntityFramew
 
   -n, --namespace    Namespace for the generated configuration classes
 
+  -s, --suffix       Suffix for the generated configuration files
+  
   -B, --no-backup    Don't keep a copy of the original DbContext file
+  
+  -r, --no-replace   Don't replace OnModelCreating event code
```

Comment:
Should probably use another name that indicates it works on `OnModelCreating`. Is this parameter even necessary? `--no-replace` is very broad.